File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
        int[] res=new int[nums.length];
        int leftSum=0;
        int rightSum=0;
        for(int i=0;i<nums.length;i++){
            rightSum+=nums[i];
        }
        for(int j=0;j<nums.length;j++){
            leftSum+=nums[j];
            res[j]=Math.abs(rightSum-leftSum);
            rightSum-=nums[j];
        }
        return res;
    }
}

File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:
        m=len(nums)
        res=[0]*m
        leftsum=0
        rightsum=0
        for n in nums:
            rightsum+=n
        for j in range(m):
            leftsum+=nums[j]
            res[j]=abs(rightsum-leftsum)
            rightsum-=nums[j]
        return res